Tools/simulation/ignition/models/OakD-Lite/model.sdf

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sdf version='1.9'>
+  <model name='OakD-Lite'>
+    <pose>0 0 0 0 0 0</pose>
+    <self_collide>false</self_collide>
+    <static>false</static>
+    <link name="OakD-Lite/base_link">
+      <inertial>
+        <pose>0.00358 -0.03 .014 0 0 0</pose>
+        <mass>0.061</mass>
+        <inertia>
+          <ixx>0.0000460804</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.0000055421</iyy>
+          <iyz>0</iyz>
+          <izz>0.0000436519</izz>
+        </inertia>
+      </inertial>
+      <visual name="OakD-Lite/visual">
+        <pose>0 0 0 0 0 0</pose>
+        <geometry>
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>model://OakD-Lite/meshes/OakDLite.dae</uri>
```

Comment:
The mesh for the OadDLite is 6MB. Is it really necessary to have such a high resolution mesh for the rgb-d camera? Just concerned that this would be part of the git tree if merged

Reply:
I foresee the removal/deprecation of all other simulators, so for our "only sensor" to be 6MB right now I don't think it's that big of a deal. It's also since the Oak-D is software configurable therefore killing the need for most other sensor requirements and can more or less be our one "in tree sensor".

Reply:
I just removed it.